applications/nftaa.md

```diff
@@ -0,0 +1,411 @@
+setting the derived account as the caller, and executing the original call.
+This mechanism ensures secure and efficient operations, allowing NFTAAs to function effectively within the blockchain
+ecosystem.
+
+<img src="https://raw.githubusercontent.com/decenzio/NFTAA-Polkadot/main/proposal/nftaa_seq_dia_proxy.svg" alt="Proposal Diagram" width="70%" height="30%">
+
+Ultimately, the proposed pallet will enable the creation of NFTAAs, setting attributes, and staking on the relay chain via
+the NFTAA.
+This innovative approach not only enhances the functionality and security of NFTs within the blockchain ecosystem but
+also opens new possibilities for their application in various business and financial contexts.
+By combining the strengths of NFTs and blockchain technology, NFTAA represents a significant step forward in the
+evolution of digital assets and their management.
+
+### Ecosystem Fit
+
+The NFTAA project aims to modernize staking and account management within the Polkadot and Substrate ecosystems through
+the innovative use of NFTs by leveraging the flexibility of the Substrate framework and Polkadot's interoperability features.
+These NFT-based accounts can own assets, participate in staking, and serve as proxies for other accounts, significantly
+enhancing security, transparency, and liquidity within blockchain networks.
+
+NFTAA targets a diverse audience, including parachain developers, dApp developers, wallet developers, end users, and
+researchers.
+Parachain developers building custom blockchain networks on Polkadot can benefit from enhanced staking and account
+management features offered by NFTAA.
+dApp developers will find NFTAA particularly useful for secure and flexible account management and staking solutions.
+Developers of cryptocurrency wallets can integrate NFTAA to provide users with advanced functionalities, thereby
+improving the user experience.
+Both individuals and enterprises looking for improved liquidity in their blockchain assets can leverage
+NFTAA for better asset management.
+Additionally, researchers and academics interested in exploring new blockchain innovations can study and expand upon the
+unique concepts introduced by NFTAA.
+
+The NFTAA project addresses several critical needs within the blockchain ecosystem.
+Utilizing NFTAA provides more methods for managing and delegating staking rights without exposing private keys.
+This approach significantly reduces the risk of unauthorized access and enhances overall security.
+Furthermore, NFTAA allows staked assets to remain liquid, enabling users to trade, lend, or use them as collateral even
+while they are staked.
+This flexibility is crucial for users who need access to their assets without compromising their staking benefits.
+The project also potentially leverages Polkadot's ability to connect multiple blockchains, ensuring seamless asset transfers and
+interactions across different networks.
```

Comment:
How exactly will this solution work in a cross chain style? As you are not the holder of the private key to the address of the keyless account you couldn't access the funds on the account with the same address on another network. This sounds like a problem to me if you were to simply transfer funds via xcm. How are you solving this?

Reply:
We are looking into and elaborating more about at cross-chain NFTs idea what is being done by other teams (e.g. Paraspell or Unique Network) but currently you are 100% right. But we will see what will future bring to us.

Reply:
Sounds good, maybe you can find a way to simply distribute these NFTs from one chain to other chains to have the effect of NFTAAs on all relevant chains. Something to figure out later.